CONTRIBUTORS.md

```diff
@@ -56,3 +56,4 @@ dmfxyz                         | `dmfxyz.eth`
 daltoncoder                    | `dontkillrobots.eth`
 0xf4ce                         | `0xf4ce.eth`
 phaze                          | `phaze.eth`
+Hari                           | `hrkrshnn.eth`
```

Comment:
Shouldn't the LHS be the GH handle?

Reply:
ah, nice. Will fix.

Reply:
Hmm, there are names and also handles. Will just use handle for now.